kombu/asynchronous/aws/ext.py

```diff
@@ -18,9 +18,50 @@ class BotoCoreError(Exception):
     exceptions = _void()
     exceptions.BotoCoreError = BotoCoreError
     AWSRequest = _void()
-    get_response = _void()
 
 
 __all__ = (
     'exceptions', 'AWSRequest', 'get_response'
 )
+
+
+def get_response(operation_model, http_response, protocol):
+    """Parses the HTTP response of an AWS operation.
+
+    This is an almost identical copy of botocore's get_response:
```

Comment:
Can the botocore implementation be aliased and operation_model.metadata['protocol'] be set to the protocol parameter?

It would be safer if the botocore implementation was used inside this function to prevent drift.

Reply:
Actually, we don't need this method anymore, and I will just remove it. Initially, I was trying to have `get_response` always parse the response as `query`. This, however, didn't work, and I decided to craft the request to match the format the operation is using, and hence we don't need our version of `get_response` and I will accordingly rename the PR.